Source/Scene/Cesium3DTileset.js

```diff
@@ -630,6 +624,24 @@ define([
         },
 
         /**
+         * A 4x4 transformation matrix that transforms the tileset's root tile.
+         *
+         * @type {Matrix4}
+         * @default Matrix4.IDENTITY
+         */
+        modelMatrix : {
+            get : function() {
+                return this._modelMatrix;
+            },
+            set : function(value) {
+                this._modelMatrix = Matrix4.clone(value, this._modelMatrix);
+                if (defined(this._root)) {
+                    this._root.updateTransform();
+                }
```

Comment:
Maybe add a comment to explain this corner case if you think we would forget later.